### DISCUSSION TITLE
What browsers are expected to work with STT/TTS?

### DISCUSSION THREAD

**Discussion by rnavarroz** [danny-avila/LibreChat#4292] [Q&A]
I could not find in the documentation on what browsers are supported for STT/TTS or the expected browsers should work.
https://www.librechat.ai/docs/configuration/stt_tts

**Comment by jquiros2** [comment]
Also looking for the same.  Understandably as there are sooo many features being worked on, it seems there is no review/summary of available options.

I set up TTS and STT using openai and it works really well.  But rather expensive, still- just testing a few phrases was $0.25.

So, going the browser route.

On a pc:  I use chrome, and setting up both "Edge" or "Browser" as the engine in librechat tts options works well, with some delay.  Then, setting cloud-based voices, there are many more options for voices.  And they work well, without delay.

Now looking for options for ios.  It appears the Edge engine on ios does not work as an option for TTS, and the browser option is still pretty bad. Enabling "Use cloud-based voices" on ios does nothing.  Tried in several browsers in ios- safari, chrome, firefox, and edge.

So... just leaving this here in case this conversation continues.

Has anyone founf any viable alternatives for great and cheap tts on ios?  It would seem like ios with popular browsers have good tts voices built in, and there should be some way to tie them into librechat.
